CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
   following flags:
   
   ```
+      --already-initialized     Specifies the PKI does not need to be created and the db does not need to be initialized. default: false
```

Comment:
does this imply `--persistent`? maybe mention this

Reply:
i'll clarify. it doesn't imply --persistent at this time, but if you don't supply --persistent it'll tear it all down on exit. so if you want to keep your pet instance. you'd have to:

* ziti edge quickstart --persistent
* (shut it down)
* ziti edge quickstart --persistent --already-initialized

Reply:
actually I think --home should imply persistence otherwise it always start in a new temp dir, right?

Reply:
if --home is not supplied it'll use a tempdir. if --home is supplied, it could default to --persistent, sure but i'm leery of adding too implicit rules around it too quickly though at this time. I'm leaning towards keeping it all explicit

Reply:
ok. you convinced me. i'll remove --persistent entirely and use --home instead. I thoght you were thinking to keep --persistent too but I can see removing the --persistent flag in favor of the user specifying it